abel/hansenlaw.py

```diff
@@ -5,169 +5,201 @@
+        .         |   *          |    *   |
+        .         |     *        | *      |
+        .         +--------      +--------+
+
+        In accordance with all PyAbel methods the image center ``o`` is
+        defined to be mix-pixel i.e. an odd number of columns, for the
+        full image.
+
 
-    :math:`f(r)` is the reconstructed image (source) function,
-    :math:`g'(R)` is the derivative of the projection (measured) function
+    For the full image transform, use the :class:``abel.Transform``.
 
-    Evaluation follows Eqs. (15 or 17), using (16a), (16b), and (16c or 18) of 
-    the Hansen and Law paper. For the full image transform, use the 
-    class :class:``abel.Transform``.
+    Inverse Abel transform: ::
 
-    For the inverse Abel transform of image g: ::
-    
-        f = abel.Transform(g, direction="inverse", method="hansenlaw").transform
-        
-    For the forward Abel transform of image f: ::
-    
-        g = abel.Transform(r, direction="forward", method="hansenlaw").transform
-        
-    This function performs the Hansen-Law transform on only one "right-side" image, 
-    typically one quadrant of the full image: ::
+      iAbel = abel.Transform(image, method='hansenlaw').transform
 
-        Qtrans = abel.hansenlaw.hansenlaw_transform(Q, direction="inverse")
+    Forward Abel transform: ::
 
-    Recursion method proceeds from the outer edge of the image
-    toward the image centre (origin). i.e. when ``n=N-1``, ``R=Rmax``, and
-    when ``n=0``, ``R=0``. This fits well with processing the image one
-    quadrant (chosen orientation to be rightside-top), or one right-half
-    image at a time.
+      fAbel = abel.Transform(image, direction='forward', method='hansenlaw').transform
 
 
     Parameters
     ----------
-    IM : 1D or 2D numpy array
-        right-side half-image (or quadrant)
+    image : 1D or 2D numpy array
+        Right-side half-image (or quadrant). See figure below.
 
     dr : float
-        sampling size (=1 for pixel images), used for Jacobian scaling
-
-    direction : string ('forward' or 'inverse')
-        ``forward`` or ``inverse`` Abel transform
+        Sampling size (=1 for pixel images), used for Jacobian scaling.
+
+    direction : string 'forward' or 'inverse'
+        ``forward`` or ``inverse`` Abel transform.
+
+    hold_order : int 0 or 1
+        First- or zero-order hold approximation used in the evaluation of
+        state equation integral. `0` (default) gives the same result as the
+        original implementation of the `hansenlaw` method. It is applicable
+        for pixel images, where the intensity of the driving function
+        (gradient for the inverse transform, original image for the forward
+        transform).  `1` assumes a linear intensity variation across
+        a 'pixel'. This may yield a more accurate transform for continuoulsy
+        varying functions.
+
+    sub_pixel_shift : float -0.35 default
```

Comment:
I think that the default value is typically listed at the end of the description of that argument.